### PR TITLE
Settings for Access and Management

### PR DESCRIPTION
**Pull request by EarlynR**

### CODE REVIEW

File: tweepy_api/settings.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#Variables that contains the user credentials to access Twitter API
+access_token = os.environ.get('TWITTER_ACCESS_KEY')
+access_token_secret = os.environ.get('TWITTER_SECRET_ACCESS_KEY')
+consumer_key = os.environ.get('TWITTER_CONSUMER_KEY')
+consumer_secret = os.environ.get('TWITTER_SECRET_CONSUMER_KEY')
+
+auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+auth.set_access_token(access_token, access_token_secret)
+
+api = tweepy.API(auth)
```
Comment (juvenfly) [inline tweepy_api/settings.py:12]: this is great! This code is actually repeated in the other script you merged, so I'll remove it on my branch and we can just import that info :)

File: tweepy_api/settings.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+connection_string = 'sqlite:///tweets.db'
+table_name = 'Lead Score'
```
Comment (juvenfly) [inline tweepy_api/settings.py:15]: We should always have a blank like at the end of a python file.

Comment (EarlynR) [reply tweepy_api/settings.py:15]: Thanks!

Comment (juvenfly) [reply tweepy_api/settings.py:15]: Butts!

**Pull request merged**